lib/srv/db/ca.go

```diff
@@ -107,7 +107,7 @@ func (s *Server) shouldInitCACertLocked(database types.Database) bool {
 		types.DatabaseTypeMemoryDB,
 		types.DatabaseTypeAWSKeyspaces,
 		types.DatabaseTypeDynamoDB,
-		types.DatabaseTypeCloudSQL,
+		types.DatabaseTypeCloudSQL, // GCP Spanner is intentionally omitted
```

Comment:
it uses system cert pool because `spanner.googleapis.com` cert is issued by a public root CA  (unlike cloud sql which can have custom CAs). I kinda just added this note for myself for when I'm running through all the places I check to make sure I got everything.

Reply:
I think this is quite useful to know this! Would you mind adding a quick note here? Others may wonder about this too.

Reply:
done 43cb579b53872d262984dd2afc3530cd27a753bf